doc/metric_spec.md

```diff
@@ -249,3 +249,42 @@ Either the value is `1`, or the line is absent altogether.
 - `disk_state`: one of `attaching|failed|negotiating|inconsistent|outdated|dunknown|consistent|uptodate`
 
 The total number of lines for this metric will be the cardinality of `name` times the cardinality of `volume`.
+
+### `ha_cluster_drbd_split_brain`
+
+#### Description
+
+This metric signal if there is a split brain occuring per resource and volume.
+Either the value is `1`, or the line is absent altogether.
+
+This metric is a special metric comparing to others, because in order to make this metric working you will need to set a drbd customer split-brain handler. Look at the end 
+
+### Labels
+
+- `resource`: the name of the resource.
+- `volume`: the volume number
+
+### Setting up the DRBD split brain hook:
+
+In order to get woring the split_brain metric:
+
+1) create hook:
+
+```bash
+#! /bin/bash
+mkdir -p /var/run/drbd/splitbrain
+echo "DRBD split-brain detected! Remember to remove the file /var/lib/drbd/drbd-split-brain-detected once the splitbrain is solved!" > /var/run/drbd/splitbrain/drbd-split-brain-detected-$DRBD_RESOURCE-$DRBD_VOLUME
+```
+
+2) copy this hook into all drbd nodes:
+```/var/lib/drbd/notify-split-brain-haclusterexporter-suse-metric.sh```
```

Comment:
rather than being so explicit and risking of repeating ourselves with the other project, why don't we refer to it directly?

e.g.
```
1) get the hook from [SUSE/ha-sap-terraform-deployments](https://github.com/SUSE/ha-sap-terraform-deployments/blob/88a2b5612e92d34976c53628ba9f33300a63bd8b/salt/drbd_node/files/notify-split-brain-haclusterexporter-suse-metric.sh)
```

Reply:
@stefanotorresi  I tought about this. On the other side I didn't want to have a broken link :grin:  if we move location. Both approaches have downsides imho.

I think having redundant data,  can be safer to broken link. But I can change it  :grin:

Reply:
we can have the links and see how it goes :grin:

Reply:
if we use the github permaurl of the file, it will never be broken ;)

Reply:
done